ecocodephp.properties

```diff
@@ -1,8 +1,8 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for PHP language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
```

Comment:
Please add text about the deprecation here.

Reply:
update done